### PR TITLE
Update july2024

### PR DESCRIPTION
**Pull request by rneher**
- this PR removes some 'empty' lineage files that were left-overs from a lineage clean-up done last year
- avoids position L:1725 due to excessive homoplasy
- adds A.D.1.8 which seemed 'left-out'

### CODE REVIEW

**Comment by rneher** [comment]
After defining the additional A.D.1 lineages, this seemed to be 'left out' and I added A.D.1.8 definition. It has 6 aa changes relative to the parent:
image

the remainder of the PR is a bit of clean-up and fixes to definition (I am not sure how the spurious lineages got in there, but the files were >8months old, might have had to do with the reorganization before the final submission).

**Comment by rneher** [comment]
The corresponding datasets are available on a preview branch for nextclade:
https://clades.nextstrain.org/?dataset-server=gh:@update-rsv

File: lineages/A.D.3.4.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ defining_mutations:
   position: 204
   state: R
 - locus: L
-  position: 559
```
Comment (AngieHinrichs) [inline lineages/A.D.3.4.yml:9]: ah, sorry I missed this typo before.

File: lineages/A.D.3.5.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ defining_mutations:
   state: I
 - locus: N
   position: 129
-  state: Y
```
Comment (AngieHinrichs) [inline lineages/A.D.3.5.yml:13]: oops, sorry I missed this too.

**Review by AngieHinrichs — APPROVED** [review]
Thanks for the cleanup.  Looks good, sorry I missed a couple defining-mut typos in #7.

**Pull request merged**